src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -588,13 +804,13 @@ def fetch_status(self, run: "PipelineRunResponse") -> ExecutionStatus:
 
     def compute_metadata(
         self,
-        execution: Any,
+        execution_arn: Any,
```

Comment:
Isn't this a `Optional[str]` now?

Reply:
Agreed with `str` but why do you think we need to add the `Optional` here?

Reply:
The first line in the function is a check `if execution_arn`, that's why I thought so

Reply:
Yeah, I saw that too, and that if condition was redundant. I am adjusting the code accordingly.